main.go

```diff
@@ -116,24 +118,34 @@ type OutputHandler interface {
 // TODO: change this into an error channel
 func reportError(d string, errmsg string, err error) {
 	status.ErrorCount.Add(1)
-	log.Printf("%s when processing %s: %s", errmsg, d, err)
+	log.Errorf("%s when processing %s: %s", errmsg, d, err)
 }
 
 func reportBundleDetails(routingKey string, body []byte, headers amqp.Table) {
-	log.Printf("Error while processing message through routing key %s:", routingKey)
+	log.Errorf("Error while processing message through routing key %s:", routingKey)
 
 	var env *sensor_events.CbEnvironmentMsg
 	env, err := createEnvMessage(headers)
 	if err != nil {
-		log.Printf("  Message was received from sensor %d; hostname %s", env.Endpoint.GetSensorId(),
+		log.Errorf("  Message was received from sensor %d; hostname %s", env.Endpoint.GetSensorId(),
 			env.Endpoint.GetSensorHostName())
 	}
 
 	if len(body) < 4 {
-		log.Println("  Message is less than 4 bytes long; malformed")
+		log.Info("  Message is less than 4 bytes long; malformed")
 	} else {
-		log.Println("  First four bytes of message were:")
-		log.Printf("  %s", hex.Dump(body[0:4]))
+		log.Info("  First four bytes of message were:")
+		log.Errorf("  %s", hex.Dump(body[0:4]))
+	}
+
+	/*
+	 * We are going to store this bundle in the DebugStore
+	 */
+	if config.DebugFlag {
+		h := md5.New()
+		h.Write(body)
+		log.Debugf("Writing Bundle to disk: %s", config.DebugStore+fmt.Sprintf("/event-forwarder-%X", h.Sum(nil)))
```

Comment:
change this to use proper path handling (just in case you're running the event forwarder on windows) - calculate the path once and use it in the debug log and the file output